content/hegel/reference/existence/development.mdx

```diff
@@ -0,0 +1,221 @@
+`being`.
+
+The second matter is that the unity of `being` and `non-being` that constitutes
+`existence` is itself in the form of `being`.
+
+```mermaid
+flowchart TB
+    subgraph being_as_process["<code>being</code>"]
+    being_a["<code>being</code>"] <---> nothing_a["<code>non-being</code>"]
+    end
+```
+
+The fact that `being` and `non-being` are in the form of `being` ensures their
+unity while also containing their difference. This unity clearly is concrete
+since it both brings together as well as holds apart divergent elements. This is
+exactly what establishes _determinateness_ for Hegel. But how does it achieve
+that? What exactly is implied in this immediate form of determinateness?
+
+### Determinateness as a Concrete Whole
+
+> The _whole_ is likewise in the form or _determinateness_ of being, since in
+> becoming being has likewise shown itself to be only a moment – something
+> sublated, negatively determined. It is such, however, _for us_, _in our
+> reflection_; not yet as _posited_ in it. What is posited, however, is the
+> determinateness as such of existence, as is also expressed by the _da_ (or
+> “there”) of the _Dasein_ (Hegel 2010, 84/21.97).
+
+This passage is tricky since since it appears that Hegel looks ahead from the
+current situation. In _our_ reflection&mdash;as concrete thinkers with more
+concepts available to us than the logic at hand&mdash;is it noted that the
+whole, or the unity, of `being` and `non-being` must equally be a _determined
+being_, no less than its constituent moments. Though this is not evident in the
+immanent development.
+
+Following this passage, Hegel appears to
+[insert a comment](/articles/hegel/reference/_annotations/textual#inline-comments)
+about the nature of positing, which serves to show that the distinction just
+made was for educational purposes rather than demonstrating anything about the
+development.
+
+> Existence corresponds to being in the preceding sphere. But being is the
+> indeterminate; there are no determinations that therefore transpire in it. But
+> existence is determinate being, something concrete; consequently, several
+> determinations, several distinct relations of its moments, immediately emerge
+> in it (Hegel 2010, 84/21.98).
+
+Hegel ends this section by stating that `existence` corresponds to `being` but
+that while the latter was indeterminate, the former is, by contrast,
+determinate. Two more key things to note in this passage is the meaning of
+"determinations transpiring" in a category and the idea that _several_
+determinations and distinctions _immediately_ emerge. From the latter,
+`existence` is regarded as something immediately complex, as was already
+understood from the preceding despite its simplicity. But exactly
+this&mdash;which relates to the former matter about "transpiring"&mdash;reveals
+the glimmers, as it were, of a movement amidst this static determinacy.
+
+#### Is the Whole an External Reflection? (Niklas)
```

Comment:
Good catch on external reflection! and thoughtful questions. My take on this issue:
Hegel is aware of the fact that the unity of being and **non-being** amounts to **determinateness**. Determinateness includes the moment of being "negatively determined"(21.98) but in the immediacy of the unity of being and non-being, there cannot be any negativity since it is simple unity, immediacy (21.97). So the problem here is that we happen to talk about a thing (determinateness) that is not posited in the concept of existence yet. In order to tackle with the problem, Hegel suggestion is like: “we happen to talk about it but be careful it is not yet derived properly.” “Any determinateness not yet posited in the concept itself belongs instead to our reflection […]” (21.97).
The following statement indicates the coming-up problem regarding immediacy and determinateness:  “Non-being thus taken up into being with **the result that the concrete whole is in the form of being, of immediacy, constitutes determinateness as such**” (21.97).  This determinateness, which is stated as the result of the simple unity, can only be the work of external reflection because the logic, so far, can only be concerned with the simple unity. In the same paragraph Hegel states: “That the whole, the unity of being and nothing, is in the one-sided determinateness of being is an external reflection but in negation, in something and other, and so forth, it will become posited.” The determinateness is the work of external reflection; hence, it can only give us **one-sided** form of being. Of course, this does not mean that all and every determinateness of being is the work of external reflection. Once the determinateness is made explicit in detail, we will see that the determinateness is the work of being.
The problem seems either with the immediacy of existence nor the simple unity of being and non-being (immediacy) but with developing a result (determinateness) from the immediacy of the unity of being and non-being.

Reply:
> Hegel suggestion is like: “we happen to talk about it but be careful it is not yet derived properly.” “Any determinateness not yet posited in the concept itself belongs instead to our reflection […]” (21.97).

But then, how about we just don't? How about we just describe what is posited, then? It seems to me that this attempt to avoid external reflection actually pits us deeper into it. 

Is the problem that we have determinateness (or seem to have it) at the _immediacy_ of existence? Because the _immediacy_ is incongruent or incompatible with determinateness (which cannot be merely immediate)? 

Trying to wrap my head around this. Your comments are very helpful and any anger in my questions aren't directed at you but at Hegel (or the logic).